recipe/magento_2_1/cache.php

```diff
@@ -18,3 +18,46 @@
         invoke('cache:clear:magento') :
         writeln('Skipped -> maintenance is not set');
 });
+
+set('cache_enabled_caches', 'all');
```

Comment:
Hi @tdgroot I like your PR, 
```suggestion
set('cache_enabled_caches', '');
```

I should not enable all caches by default, for some reasons, maybe some projects don't should have all caches enabled.

In your `deploy.php` you should decide what kind of caches you want enable.

Reply:
IMHO, all caches should be enabled by default. If you do not want to enable some caches, you can still customise this in your `deploy.php`.

Reply:
In this case you don’ have the possibility to disable caches in your deploy.php.

And I think that deployer not should be responsible to enable caches of application by default.

We can add the function and add to deploy.php to all, I think works for me.

We have several projects that some caches are not enable, for example some amasty caches.

Greetings!

Reply:
@osrecio @sprankhub Thank you for getting back to this changeset.

I propose the following:

1. In the recipe, caches are disabled by default.
2. In the `deploy.php` template, all caches are configured to be enabled.

Reply:
Cool, if you do this approach I will merge ASAP